README.md

```diff
@@ -73,6 +73,11 @@ Template invalid!
 `--only-guess-parameters <param names>`: Only guess the provided parameters, and disable the guessing of all others without Defaults. A critical error will be raised for missing parameters, as above. e.g.
  - `--only-guess-parameters InstanceType,Memory`
 
+`--custom-resource-attributes <attribute values>`: Provide a list of comma-separated key=value pairs of resource attributes using either resource-type notation (e.g. `AWS::CloudFormation::
+CustomResource.SomeAttribute`) or logical-name notation (e.g. `SomethingPretty.SomeAttribute`) and their expected values, to use when validating your template. If a custom resource attribute is not specified here, `cfn-lint` will guess a mock value. e.g.
+  - `--custom-resource-attributes AWS::CloudFormation::
```

Comment:
This doesn't format properly, is the bullet point is in the wrong place?

Reply:
There are a few others which don't format correctly. Leave this and I will clean it up afterwards